### ISSUE TITLE
missing some commands on win installation

### ISSUE THREAD

**Issue by paolobettotti** [LFPy/LFPy#276]
After installing LFPy  and trying to run the basic scritp shown in https://lfpy.readthedocs.io/en/latest/readme.html#requirements
LFPy fails to import LineSourcePotential. Importing Cell and Synapse raise no error.

Neuron (7.8.1) is installed and report no error after running: import neuron; neuron.test()

System config:
Win10 with Anaconda 1.10.0
python            3.8.3           he1778fa_2
scipy               1.5.2           py38h14eb087_0
numpy            1.19.2         py38hadc3359_0
matplotlib       3.3.2           0
mpi4py           3.0.3           pypi_0    pypi
hdf5               1.10.4         h7ebc959_0
meautility       1.4.8           pypi_0    pypi
lfpykit             0.2             pypi_0    pypi
lfpy                 2.1.2          pypi_0    pypi
jupyterlab       2.2.6          py_0

all packages installed in the base env
both LFPy and LFPykit installed cloning git repositories

**Comment by espenhgn** [comment]
Hi @paolobettotti , 
Thanks for letting us know. Does the `LineSourcePotential` class exist in LFPykit? What happens if you issue:
`python -c "import lfpykit; print(lfpykit.LineSourcePotential)"`

Full disclaimer: Neither LFPy or LFPykit has been systematically tested on Windows as I (or the other main LFPy devs) don't have Windows machines.

**Comment by paolobettotti** [comment]
yes, thanks. Reading old issues I noticed that you splitted these commands on flpykit.

Thanks,
P.

**Comment by espenhgn** [comment]
`from LFPy import LineSourcePotential` should still work. But the class is just imported from LFPykit as an LFPy class. 

We're changing things up a bit for the upcoming LFPy-2.2 release in our master branch, but the git tag "v2.1.2" or pip release "LFPy==2.1.2"  should behave as before.

**Comment by paolobettotti** [comment]
import from LFPy (2.1.2) does not work but from lfpykit it does.

**Comment by espenhgn** [comment]
It's not supposed to work in v2.1.2 ;-)
We have been adding LFPykit as an LFPy dependency in the 2.2 development branch (current master branch). Sorry for the confusion. This new version of LFPy then adds a few new classes from LFPykit that wasn't there before (`PointSourcePotential, LineSourcePotential, CurrentDipoleMoment` etc.) and change some syntax. 

If you still have problems I need to create a windows virtual machine or something for testing - in principle I don't know of anything that should fundamentally break compatibility with Windows with the possible exception of automatic compilation of the `sinsyn.mod` file which is only being used by the test suite (in `<LFPy-sources>/LFPy/tests/`). On macos/linux the `nrnivmodl` script is run automatically, but on Windows one has to use `mknrndll.exe` or something like that.

**Comment by paolobettotti** [comment]
many thanks for your kind availability, but on need to test it on VM as the calls to forward models importing them from lfpykit work perfectly!

**Comment by espenhgn** [comment]
Hi again @paolobettotti ; 
For Windows, I just merged PR #284 which should call `mknrndll` automatically from LFPy's `setup.py` file if `nrnivmodl` is not found. I tested the build on a Windows 10 VM with miniconda, NEURON 7.8.1 installer from https://neuron.yale.edu, Microsoft MPI v10.x. 

NEURON must be allowed to define environment variables (this is activated by default when installing it). 
The path to MPI must also be added to Windows PATH, see https://stackoverflow.com/questions/54615336/how-to-install-mpi4py-on-windows-10-with-msmpi

The PR doesn't fix any potential issues that may arise with LFPy's dependencies like NEURON or mpi4py, but hopefully a conda install procedure like the below should now work: 
```
conda config --add channels conda-forge
conda config --set channel_priority strict
conda create -n lfpy python=3.8 pip numpy scipy matplotlib h5py meautility lfpykit c-compiler cxx-compiler
conda activate lfpy
pip install git+https://github.com/LFPy/LFPy@master#egg=LFPy
python -c "import LFPy; LFPy.run_tests()"
```
